apps/omg_watcher/lib/exit_processor.ex

```diff
@@ -298,6 +309,27 @@ defmodule OMG.Watcher.ExitProcessor do
     {:reply, canonicity_result, state}
   end
 
+  def handle_call({:create_challenge, Utxo.position(blknum, txindex, oindex) = exiting_utxo_pos}, _from, state) do
+    with spending_blknum_response = OMG.DB.spent_blknum({blknum, txindex, oindex}),
+         ife_response = Core.get_ife_based_on_utxo(exiting_utxo_pos, state),
```

Comment:
why ```=``` and not ```<-```? 
then you can remove (if catch {:ok, val})
```
def ensure_challengeable({error,errno}_,_)
def ensure_challengeable(_,{:error,errno},_)
def ensure_challengeable(_,_,{:error, error})
```

Reply:
Because I need only 2 out of 3 variables to be valid. By variables I mean `
spending_blknum_response` , `ife_response`, `exit_response`